src/controller.py

```diff
@@ -6,12 +6,3 @@ def welcome(self):
     def soil_moisture(self, plant_class):
         plant = plant_class()
         return plant.soil_moisture()
-
```

Comment:
Do we still need a controller class?

Reply:
No! We can probably remove it from the app!